lib/channel.rb

```diff
@@ -0,0 +1,74 @@
+require 'httparty'
+require 'awesome_print'
+require 'dotenv'
+Dotenv.load
+
+class Channel
+  def self.get_channels
+    url = 'https://slack.com/api/channels.list'
+    query_parameters = {
+      token: ENV['SLACK_API_TOKEN']
+    }
+    
+    response = HTTParty.get(url, query: query_parameters).to_s
+    response = JSON.parse(response)
+
+    return response["channels"]
+  end
+  
+  def self.list_channels(channels)
+    channels.each do |channel|
+      puts "Channel name: #{channel["name"]}"
```

Comment:
You are also mixing the presentation parts of the program with your business logic.  It's advisable to put the parts that deal with data and application logic separate from code that presents an interface to the user.